src/css/application.css

```diff
@@ -131,73 +134,107 @@ body {
   background-color: var(--color-red);
 }
 
-.top-bar__logo-container > svg {
-  height: 3em;
+.top-bar__logo-container {
+  margin: 0 0.4em 0 0.2em;
+  min-width: 1.6em;
+  transform: scale(1.8);
+  width: 1.6em;
 }
 
 .top-bar__wordmark-container {
   position: relative;
-  margin-left: -0.25em;
+  margin: 0 1em 0 0.5em;
+  min-width: 6.5em;
+  width: 6.5em;
```

Comment:
Wondering about the `min-width` and `width` together again?

Reply:
Done. Handled differently.